src/components/VAudioTrack/VGlobalAudioTrack.vue

```diff
@@ -107,7 +108,11 @@ export default defineComponent({
 
     const status = ref('paused')
     const currentTime = ref(0)
-    const audioDuration = ref(null)
+    const duration = defaultRef(() => {
+      if (typeof props.audio?.duration === 'number')
```

Comment:
Can this ever be `NaN`? Would it be safer to add a check for `NaN`, too?

Reply:
`props.audio` is the Audio result object which doesn't get modified by the frontend once it's decoded.

NaN isn't a valid JSON value so I don't think it could ever end up there:

```
> JSON.stringify({'a': NaN})
'{"a":null}'
```